wireguard-go-rust-wrapper/wireguard-go/controlfns_unix.go

```diff
@@ -0,0 +1,35 @@
+//go:build !windows && !linux && !wasm
```

Comment:
We currently only support wg-go on windows. Is this file required anyways, or was there an otherwise active decision to keep files for unix compat?

Reply:
Same question for `controlfns_linux.go`

Reply:
I've decided to copy all versions of controlfns_* as is to make it simpler when comparing to the upstream (and also when building locally on different OSes)